Cython/Compiler/Symtab.py

```diff
@@ -158,13 +158,17 @@ class Entry(object):
     # is_fused_specialized boolean Whether this entry of a cdef or def function
     #                              is a specialization
     # is_cgetter       boolean    Is a c-level getter function
+    # unambiguous_import_path     Either None (default), False (definitely can't be determined)
```

Comment:
Do you mean known standard library imports, maybe?

Reply:
Yeah I'm happy with that as a name. Provided we no-one ever tries to use this mechanism on a non-standard library module like Numpy in future